src/tests/test_await_jsproxy.py

```diff
@@ -73,6 +73,33 @@ async def test():
             """
         )
 
+def test_await_error(selenium):
+    selenium.run_js(
+        """
+        async function js_raises(){
+            throw Error("This is an error message!");
```

Comment:
Oh wow I was just using `Error()` without a `new` and it is causing some really weird low level exception. I think this means we need a test with both versions: `throw Error()` without `new` is a logic error, but it shouldn't completely crash the Pyodide runtime.

Reply:
Not sure exactly, but since `throw` can be used to throw any object/expression in js, it will just throw an object `Error("This is an error message!")`. 
I did the following experiment in the JS console:
```
obj = Error("This is an error message!")
typeof obj 
// "object"
```

I think if you convert the object into a string then make a Python exception according to my other comment, this type of wired exception might go away.